pkg/signature/kms/hashivault/client.go

```diff
@@ -246,7 +234,21 @@ func (h *hashivaultClient) fetchPublicKey(_ context.Context) (crypto.PublicKey,
 }
 
 func (h *hashivaultClient) public() (crypto.PublicKey, error) {
-	return h.keyCache.Get(cacheKey)
+	var lerr error
+	loader := ttlcache.LoaderFunc[string, crypto.PublicKey](
```

Comment:
What do you think about setting the loader on initialization `ttlcache.WithLoader`, like what's done in the gcp client?

Reply:
I've changed the gcp client to also use a local loader.

Without a local error, errors while fetching keys would be silently discarded,
making diagnoising misconfiguration and other issues much harder than
necessary.

Reply:
Thanks for clarifying that. Using a local one sounds good then.